tdm-be/src/lib/codes.ts

```diff
@@ -0,0 +1,13 @@
+// User input error
+export const ERROR_CODE_WRAPPER_BAD_USER_INPUT = '40';
+export const ERROR_MESSAGE_WRAPPER_BAD_USER_INPUT = `Le convertisseur a rencontrais une erreur inattendue. Code d'erreur ${ERROR_CODE_WRAPPER_BAD_USER_INPUT}`;
+
+// Server Error
+export const ERROR_CODE_UNEXPECTED_ERROR = '50';
+export const ERROR_MESSAGE_UNEXPECTED_ERROR = `Le convertisseur a rencontrais une erreur inattendue. Code d'erreur ${ERROR_CODE_UNEXPECTED_ERROR}`;
+
+export const ERROR_CODE_FILE_SYSTEM_ERROR = '51';
+export const ERROR_MESSAGE_FILE_SYSTEM_ERROR = `Le convertisseur a rencontrais une erreur inattendue. Code d'erreur ${ERROR_CODE_FILE_SYSTEM_ERROR}`;
+
+export const ERROR_CODE_WRAPPER_UNREACHABLE_ERROR = '52';
+export const ERROR_MESSAGE_WRAPPER_UNREACHABLE_ERROR = `Le convertisseur a rencontrais une erreur inattendue. Code d'erreur ${ERROR_CODE_WRAPPER_UNREACHABLE_ERROR}`;
```

Comment:
```suggestion
export const ERROR_MESSAGE_WRAPPER_UNREACHABLE_ERROR = `Le convertisseur a rencontré une erreur inattendue. Code d'erreur ${ERROR_CODE_WRAPPER_UNREACHABLE_ERROR}`;
```

Reply:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** Error message could be more accurate: the wrapper was unreachable.‏